src/lib/sticky-header/sticky-header.spec.ts

```diff
@@ -0,0 +1,180 @@
+import {async, ComponentFixture, TestBed, fakeAsync, tick} from '@angular/core/testing';
+import {Component, DebugElement, ViewChild} from '@angular/core';
+import {StickyHeaderModule, CdkStickyRegion, CdkStickyHeader,
+  STICKY_HEADER_SUPPORT_STRATEGY} from './index';
```

Comment:
If you run the "optimize imports" command in webstorm it will format these imports for you.